### DISCUSSION TITLE
Updating to latest release broke my setup

### DISCUSSION THREAD

**Discussion by wjschmitt** [aaronwmorris/indi-allsky#1433] [General]
My allsky setup has been running flawlessly for the past four months.  I have a raspberry pi with HQ camera running indiserver and a proxmox VM running the indi-allsky service.  On the VM, I point indi-allsky to the indiserver IP and the camera selected is indi_pylibcamera.  This was working fine until I decided to upgrade to the latest release 2024.05.1, which I did on both the VM and the Pi.
Now when I start indi-allsky service on the VM, it shows status: active (running).  On the Pi, when I start indiserver -v indi_pylibcamera, it shows the VM trying to connect and then it says read EOF and shutdown complete.

Where can I look to try and track down the problem?  Did I do something wrong in the upgrade and set up?  Thanks.

**Comment by aaronwmorris** [comment]
Have you just tried fully rebooting?

**Reply by wjschmitt** [reply]
OK that did the trick.  I was sure that I had done that but maybe not.  I also have to be careful when restarting everything because I have to make sure that indi-allsky doesn't start on the Pi, only indiserver.  And the indi-allsky.timer has to be disabled so it doesn't start the service.  Same on the VM - stop the indiserver here and only start indi-allsky.

Thanks for helping out, Aaron.

**Comment by wjschmitt** [comment]
Just to continue this discussion  -  where can I find out the installed version/release info?  I have two identical setups - same Pi, same HQ camera, both running indiserver.  Under System, both list the Release as indi_v2024.09.1.  However there are differences in configuration options.  I tried to make all option settings the same on each system, but options are displayed in different places or missing/added in one setup or the other.  The problem is that one setup works good and one does not.  For instance, I can normally get star trails to generate on one setup, but they won't generate on the other for the same night.  I want to maybe update the non-working setup to be exactly the same as the working one, but I don't know which version/release is better.  I tried looking through the support info and comparing but I couldn't see much differences.  

I don't know if the working setup is the latest or not, and I do not want to just update the working version to the latest in case the non-working one is the latest.  What do I do?

**Reply by omeganon** [reply]
Assuming you did a git checkout to download the code, probably the best way would be to go into your indi-allsky directory and see what the most recent commit was for each install using git log.

e.g. this install is current up to February 21 2025 --

```
admin@allsky:~/indi-allsky $ git log | head -n 20
commit d265bcf64bf78f66bd91d37a2d5543f7a784e951
Author: Aaron W Morris <aaron@aarmor.net>
Date:   Fri Feb 21 12:35:46 2025 -0600

    add lens offset rotation option (work in progress)

commit cc5a467dd89bf5fdd942f8acaa78c640fefd0fce
Author: Aaron W Morris <aaron@aarmor.net>
Date:   Fri Feb 21 10:28:27 2025 -0600

    add config option to maintain image size with rotation

commit ebf04860c4cd7e44c5ff743900cd44f114f00d61
Merge: ce4299f1 0a808768
Author: Aaron W Morris <aaron@aarmor.net>
Date:   Thu Feb 20 18:27:14 2025 -0600

    Merge pull request #1907 from aaronwmorris/dev

    scale video to fit in visible area
```

**Reply by wjschmitt** [reply]
Excellent.  Thank you.  Now I'm not sure what to do.  Neither system is the latest.  And the working system is older than the non-working system.  I guess I will update them both to get them on the same release and see what happens.

**Reply by wjschmitt** [reply]
Just to update this thread - I went ahead and updated all systems - both Pi's and both VMs - to the latest, and now everything is working.
